datamodel/app/view/maintenance_views.py

```diff
@@ -177,7 +177,7 @@ def vw_tww_channel_maintenance(connection: psycopg.Connection, extra_definition:
         , {ch_cols}
         {extra_cols}
       FROM tww_od.re_maintenance_event_wastewater_structure mw
-         INNER JOIN tww_app.vw_tww_channel ch ON ch.obj_id = mw.fk_wastewater_structure
+         INNER JOIN tww_app.vw_tww_channel ch ON ch.ch_obj_id = mw.fk_wastewater_structure
```

Comment:
@sjib I did not look too deep but if a view is called channel, the channel attributes will not have a `ch_` prefix so in your cas it will be `obj_id` instead of `ch_obj_id`. Prefix are used to distinguish attributes coming from other elements.